scripts/backport-siw-next.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# scripts/backport-siw-next-beta.sh
+
+# the temporary backport branch, e.g., backport-9012f7d4
+FIX_BRANCH="backport-$(git rev-parse --verify 0.1 | cut -c-8)"
+
+# the target branch
+BRANCHES_TO_BACKPORT="0.1"
+
+# obviously
+GIT_REPO="okta-signin-widget"
+
+# merge
+git fetch origin && \
+git checkout 0.1 && \
+git checkout -b $FIX_BRANCH && \
+git merge 0.0 && \
```

Comment:
How does the script behave on a merge conflict in this step?

Reply:
Good question.

It should exit with code 0 (success) if there were no issues. It should ask you to fix merge conflicts if any (non-zero exit code?) preventing you from proceeding to push.

But I think the script shouldn't assume the merge will be conflict-free.
Let me break it up into "merge" and "push" sections so you can resume after fixing conflicts.